dags/utils/landsat_scenes_identifying_logic.py

```diff
@@ -148,9 +149,14 @@ def build_asset_list(scene):
         )
 
         if not response.get("Contents"):
-            raise Exception(
-                f"Error Listing objects in S3 {USGS_S3_BUCKET_NAME} folder {folder_link}"
+            notify_email(
+                task_name=f'Landsat Identifying - {scene["Date Product Generated L2"]}',
+                warning_message=f"Error Listing objects in S3 {USGS_S3_BUCKET_NAME} -"
+                f" folder {folder_link} - response {response}",
             )
+            # raise Exception(
+            #     f"Error Listing objects in S3 {USGS_S3_BUCKET_NAME} folder {folder_link}"
+            # )
 
         mtl_sr_st_files = {}
         for obj in response["Contents"]:
```

Comment:
This is the section I'm wandering. Should you also have a line to skip process the response further?

Reply:
you are totally right ... fixed